api/v1/firewall_types.go

```diff
@@ -79,6 +79,8 @@ type Data struct {
 	EgressRules []EgressRuleSNAT `json:"egressRules,omitempty"`
 	// FirewallNetworks holds the networks known at the metal-api for this firewall machine
 	FirewallNetworks []FirewallNetwork `json:"firewallNetworks,omitempty"`
+	// DisableSuricataIDS specifies if we need to enable IDS on the firewall machine
+	DisableSuricataIDS bool `json:"disableSuricataIDS,omitempty"`
```

Comment:
I would not put any product names into the firewall resources, also it should be disabled by default if nothing was set, so `EnableIDS` would be more approriate, WDYT ?

Reply:
Yep, i also thought so initially. But currently we have a flag `enable-IDS` that is set to true by default. It's used for writing IDS stats to Firewall resource. Should be rid of this flag or change it's default value?(potentially, breaking backwards compatibility)

Reply:
We should remove the flag from main.go, and only rely on the Flag set in the firewall resource. The service will be started by systemd even if a unknown flag is given (needs to be tested).

Reply:
Done